modules/tinymce/src/themes/silver/main/ts/ui/general/UiFactory.ts

```diff
@@ -72,7 +72,7 @@ const factories: Record<string, FormPartRenderer> = {
   urlinput: make((spec, backstage) => renderUrlInput(spec, backstage, backstage.urlinput)),
   customeditor: make(renderCustomEditor),
   htmlpanel: make(renderHtmlPanel),
-  imagetools: make((spec, backstage) => renderImageTools(spec, backstage.shared.providers)),
+  imagepanel: make((spec, _backstage) => renderImagePanel(spec)),
```

Comment:
```suggestion
  imagepanel: make(renderImagePanel),
```
We do this for `htmlpanel` and `customeditor` above, so we should probably do the same here :shrug:

Reply:
it runs a bit of a risk for future refactoring, but I guess that's fine.